docs/api/spaces-management/put.asciidoc

```diff
@@ -34,6 +34,10 @@ experimental[This functionality is *experimental* and may be changed or removed
 `color`:: 
   (Optional, string) Specifies the hexadecimal color code used in the space avatar. By default, the color is automatically generated from the space name.
 
+`imageUrl`:: 
+  (Optional, string) Specifies the data-url encoded image to display in the space avatar. If specified, `initials` will not be displayed, and the `color` will be visible as the background color for transparent images.
+  For best results, your image should be 64x64. Images will not be optimized by this API call, so care should be taken when using custom images.
```

Comment:
To make the second sentence active, I would change it to "Images are not optimized by this API call."

Could we specify what "so care should be taken when using custom images" means? Or maybe change it to "Images are not optimized by this API call. Use custom images sparingly."

Reply:
> Could we specify what "so care should be taken when using custom images" means? Or maybe change it to "Images are not optimized by this API call. Use custom images sparingly."

How about something like this?
"Images are not optimized by this API call. Consider using the Spaces Management UI to have your images properly resized for efficient storage and display."